content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+After initiating a chain by providing the first participant with a set of documents to read on their own, later participants were left to talk freely in successive dyads, and the whole session was recorded on film for subsequent analysis.
+An analogous change of setup was operated by @caldwell_studying_2008 [-@caldwell_experimental_2008] who investigated the cumulative aspect of the evolution of building techniques in a series of experiments asking participants to construct spaghetti towers or paper planes, later evaluated by their height and flight distance.
+A crucial point in those setups, Sterelny notes [-@sterelny_cultural_2017, n. 12], is that participants could observe the preceding generation during their experimenting and building, meaning they had access to rich context for the learning phase of transmission.
+Taking another step to embrace interaction with a setup where participants had to repeatedly build Lego cars in pairs (though without transmission), @mcgraw_cultures_2014 take the complete joint interaction to be their object of study and propose to use the constructions resulting from interacting dyads as a trace of the processes that took place in the interaction.
+By looking at the characteristics of the cars built by participants, the authors claim to present "methods for discerning, and quantifying, schema-like intersubjective understandings in material form" [@mcgraw_cultures_2014, p. 4; see also @mitkidis_building_2015; and @wallot_beyond_2016], effectively integrating critiques that I discuss below in @sec:intro-review-criticisms.
+
+The works reviewed here show that there is much room for exploration, on one side, of the methodological choices in transmission chain experiments (some of which will turn out to be more important than others in the trends observed), and on the other, of the theoretical background that sustains a given study.
+Such experiments have therefore much to bring to underlying theoretical debates.
+
+
+#### Language change: Experimental semiotics and Iterated learning
+
+\add{tamariz-experimental-2017}
+
+A related strand of research has developed similar methods to study the emergence and evolution of language.
+A central experimental paradigm is that of iterated learning, which resembles a transmission chain for artificial languages where participants learn a communication system at each step:
+a first group of participants must learn to use a simple artificial language (e.g. an artificial vocabulary for naming a range of objects), after which a second group of participants must learn that language through some interaction with or transmission from the first group.
+The process is then iterated over successive generations, leading to the evolution of the artificial language initially introduced.
+A related setup used in experimental semiotics surfaces the evolution of interaction without transmission across generations.
```

Comment:
### Camille
"A related setup used in experimental semiotics surfaces the evolution of interaction without transmission across generations." – surface??

Reply:
Done.